deploy/policy/load_policies.sh

```diff
@@ -35,6 +35,7 @@ done
 conjur variable values add secrets/test_secret "some-secret"
 conjur variable values add "secrets/var with spaces" "some-secret"
 conjur variable values add "secrets/var+with+pluses" "some-secret"
+conjur variable values add "secrets/umlaut" "some-secret"
```

Comment:
Do you not need an umlaut in this secret value? Or is this just a placeholder that gets replaced later on?

Reply:
It's a [placeholder](https://github.com/cyberark/secrets-provider-for-k8s/blob/main/deploy/policy/load_policies.sh#L32-L34) that's replaced when a test case runs [[test case](https://github.com/cyberark/secrets-provider-for-k8s/pull/500/files#diff-d88d31cb2f68e87775b2b9c5ee06811ba27e2392fdce07440f3a59b7211ff675R8)][[CLI variable load](https://github.com/cyberark/secrets-provider-for-k8s/blob/main/deploy/utils.sh#L504)].